infra/modules/azure_naming_convention/README.md

```diff
@@ -1,5 +1,36 @@
 # DX - Common Variables Validation
 
+This module generates output values that follow the naming conventions used in DX. It takes the following inputs:
```

Comment:
This will be resolved with multiple environments, look at this [PR](https://github.com/pagopa/dx/pull/179), in this way you can configure different environments, so with other `app_names` or without `domains`